mne/viz/utils.py

```diff
@@ -1184,19 +1205,27 @@ def plot_sensors(info, kind='topomap', ch_type=None, title=None,
                 if pick in value:
                     colors[pick_idx] = color_vals[ind]
                     break
-    if kind == 'topomap':
+    if kind in ('topomap', 'select'):
         pos = _auto_topomap_coords(info, picks, True)
 
     title = 'Sensor positions (%s)' % ch_type if title is None else title
     fig = _plot_sensors(pos, colors, bads, ch_names, title, show_names, axes,
-                        show)
+                        show, kind == 'select')
```

Comment:
Actually it is possible by modifying the list instead of reassigning. I'll make this happen.